Modules/xSharePoint/DSCResources/MSFT_xSPOutgoingEmailSettings/MSFT_xSPOutgoingEmailSettings.psm1

```diff
@@ -0,0 +1,94 @@
+function Get-TargetResource
+{
+	[CmdletBinding()]
+	[OutputType([System.Collections.Hashtable])]
+	param
+	(
+		[parameter(Mandatory = $true)] [System.String] $WebAppUrl,
+		[parameter(Mandatory = $true)] [System.String] $SMTPServer,
+		[parameter(Mandatory = $true)] [System.String] $FromAddress,
+		[parameter(Mandatory = $true)] [System.String] $ReplyToAddress,
+		[parameter(Mandatory = $true)] [System.String] $CharacterSet
+		
+	)
+
+	Write-Verbose -Message "Retrieving outgoing email settings configuration "
+
+	$result = Invoke-xSharePointCommand -Arguments $PSBoundParameters -ScriptBlock {
+		$params = $args[0]
+		
+		$webApp = $null
+		if([string]::IsNullOrEmpty($params.WebAppUrl)){
```

Comment:
That's fine we can include that in the documentation. I've been working on a way to generate the wiki pages and inbuilt module help from the same source so it will be easy for people to understand how it works.